README.md

```diff
@@ -6,7 +6,72 @@ Depending on the source environment, either the Webserver Plugin or the `AstroMi
 
 **Note:** In order to use the Starship utilities, the source Airflow environment must be running Airflow 2.x
 
+
 Choosing the right package
 ------------
 - The [AstroMigrationOperator](https://github.com/astronomer/starship/tree/master/astronomer-starship-provider) should be used if migrating from a Google Cloud Composer 1 (with Airflow 2.x) or MWAA v2.0.2 environment. These environments do not support webserver plugins and will require using the `AstroMigrationOperator` to migrate Connections, Variables, and Environment Variables. 
 - The [Starship Plugin](https://github.com/astronomer/starship/tree/master/astronomer-starship) should be used for migrating from all other environments including Goocle Cloud Composer 2, MWAA v2.2.2, OSS, and Astronomer Software/Nebula instances.
+
+
+Sending connections to the a Secrets Store
+-----------------
+----
+### AWS Secrets Manager 
+To export to AWS Secrets manager you need to set the following Environment variables
+
+    SECRETS_BACKEND_TYPE=AwsSecretsManager
+    AWS_ACCESS_KEY_ID=<Your AWS Access KEY>
+    AWS_SECRET_ACCESS_KEY=<Your AWS Secret Key>
+    AWS_DEFAULT_REGION=<Your desired AWS Region>
```

Comment:
this should all be gotten from the Airflow conn, which we should ask them to have setup, IMO

Reply:
_also_ potentially even ask them to utilize the normal `AIRFLOW_SECRETS_BACKEND_KWARGS`, and just get the info from that dict. Dunno if that'd have a chance to break things - but 🤷 worthwhile using "the normal mechanisms" than making new ones